book/website/community-handbook/newsletters/newsletters-process.md

```diff
@@ -0,0 +1,113 @@
+
+5. **Tips & Tricks for new contributors**: this includes any resource in the project that can make new members learn ways to engage, identify paths to get started as contributors and find relatable contents like impact stories of existing members, contributor's profiles or other community-related aspects.
+
+6. **Acknowledgments and celebrations section**: this is the place to give shout outs to our members who have helped the project or others in some ways, celebrate personal milestones and highlights from the community members.
+
+This is also a place to share tweets from the community or mention other online interactions such as posts from recent meetings where someone talked about _The Turing Way_.
+
+The newsletter should focus more on the contributing and new members, and highlight only noteworthy content from _The Turing Way_ core members.
+
+### Collect images associated with the news item**
+
+Following the recommendations on {ref}`style guide for community<ch-style-guide>` and {ref}`style guide for newsletters<ch-newsletters-style>` for using images, collect a few (zero to maximum 2 per section) images.
+Make sure that these images are available under a free license (like CC-BY), collected with the link of their sources, and named clearly as suggested in the style guide.
+
+For the twitter mentions, there is no set number limit but recommended to use maximum 6 tweets, edited together in one image (explained in the newsletter's style guide).
+
+### Write about each news item
+
+Based on the bullet points collected for each news item, create 1-2 small paragraphs using the recommendations in the style guide for the language and format described in the next subchapter.
+
+Provide links when useful, give credits fairly to the community members who might be associated with the news item and end the paragraph with a sentence and link to more information.
+
+### Proofreading your draft
+
+Before sharing your draft to get it reviewed, you should do a proofread for grammar and typo.
+Online an app like [Ginger Grammar Checker](https://www.gingersoftware.com/grammarcheck), [Grammarly](https://app.grammarly.com) free version, [GrammarCheck](https://www.grammarcheck.net/editor/) or [Reverso Speller](https://www.reverso.net/spell-checker/english-spelling-grammar/) can help creect any grammatical and spelling errors.
+
+You should also double-check to make sure that the links mentioned in the draft are not broken.
+You can use online tools such as [W3C link checker](https://validator.w3.org/checklink) or free version of [Dr. Link Check](https://www.drlinkcheck.com/).
+
+### Updating your draft in the online repository
+
+If you have drafted your newsletter in a local branch, add all the images mentioned in the newsletter and create a PR.
```

Comment:
Does this need some more info or a link where the images should be uploaded or is that clear to people?

Reply:
A mentioned in the style guide, maybe add a heads up here?